homeassistant/components/automation/event.py

```diff
@@ -29,12 +29,12 @@ def async_trigger(hass, config, action):
     event_type = config.get(CONF_EVENT_TYPE)
     event_data = config.get(CONF_EVENT_DATA)
 
-    @asyncio.coroutine
+    @async_safe
```

Comment:
I realized there is a proper name used for this in asyncio code, ``callback``. Multiple asyncio functions and classes take callbacks instead of coroutines, so for consistency we should probably name this ``@callback``. That way its also obvious how to schedule it from the asyncio docs.